### DISCUSSION TITLE
About Semi-supervised Medical Image Segmentation

### DISCUSSION THREAD

**Discussion by JohnMasoner** [Project-MONAI/MONAI#4665] [Ideas]
Hey, 

I've been doing some work on semi-supervised segmentation recently, and I found that MONAI does not implement a semi-supervised loss function, which I think could be important and helpful.(It may also include some Network architectures and etc, In short, it is something related to semi-supervision.)

Can I contribute to the code after I finish my work? Does MONAI need it?

Looking forward to your reply.
Thanks.

**Comment by Nic-Ma** [comment]
Sure, welcome your contribution!

Thanks in advance.
